src/xbps/repositories/index.md

```diff
@@ -8,7 +8,16 @@ which may also be signed.
 Note that, while local repositories do not require signatures, remote
 repositories *must* be signed.
 
-## Official Repositories
+## The main repository
+
+The locations of the main repository in relation to a base [mirror
+URL](./mirrors/index.md) are:
+
+- glibc: `/current`
+- musl: `/current/musl`
+- aarch64 and aarch64-musl: `/current/aarch64`
+
+## Subrepositories
```

Comment:
Is subrepos the correct name for them?

Reply:
That term is used on the current usage/xbps page on the Web site, so i assume so. :-)

Reply:
Fair enough :p